service_container/service_subscribers_locators.rst

```diff
@@ -122,6 +122,24 @@ It is also a callable and a countable::
 
     return $handler->handle($command);
 
+Because the service locator implements the
```

Comment:
I wonder if we should move this to the above paragraph where we explain what `ServiceLocator` implements?